olp-cpp-sdk-dataservice-read/src/StreamLayerClientImpl.cpp

```diff
@@ -21,23 +21,142 @@
 
 #include <olp/core/cache/DefaultCache.h>
 #include <olp/core/client/OlpClientSettingsFactory.h>
+#include <olp/core/logging/Log.h>
+#include <olp/core/thread/TaskScheduler.h>
+
+#include "ApiClientLookup.h"
+#include "Common.h"
+#include "generated/api/StreamApi.h"
 
 namespace olp {
 namespace dataservice {
 namespace read {
 
+namespace {
+constexpr auto kLogTag = "StreamLayerClientImpl";
+const std::string kSubscriptionModeParallel = "parallel";
```

Comment:
Why are these string and not auto?

Reply:
Since boost::optional. Not the case now.